libs/books/data-access/src/lib/+state/reading-list.effects.spec.ts

```diff
@@ -43,4 +46,55 @@ describe('ToReadEffects', () => {
       httpMock.expectOne(API_PATH.READING_LIST).flush([]);
     });
   });
-});
+
+  describe('markBookAsFinished$', () => {
+    it('should dispatch confirmedMarkBookAsFinished action when API returns success response', done => {
+      actions = new ReplaySubject();
+      const item = {
+        ...createReadingListItem('A'),
+        finished: true,
+        finishedDate: '2021-03-03T00:00:00.000Z'
+      };
+      actions.next(ReadingListActions.markBookAsFinished({ item }));
```

Comment:
You can move Line no 52 to 58 in before each block as it's used in both test cases.